ingestion/src/metadata/ingestion/ometa/mixins/ingestion_pipeline_mixin.py

```diff
@@ -105,3 +105,19 @@ def get_pipeline_status_between_ts(
         if resp:
             return [PipelineStatus.parse_obj(status) for status in resp.get("data")]
         return None
+
+    def get_ingestion_pipeline_by_name(self, name) -> Optional[IngestionPipeline]:
+        """
+        Get pipeline statues based on name
+
+        Args:
+            name (str): Ingestion Pipeline Name
+        """
+        resp = self.client.get(
+            f"/services/ingestionPipelines?fields=owner,pipelineStatuses&service={name}",
+        )
```

Comment:
can we have a mixin method available?
increase our python sdk coverage to fetch more things from backend
win-win situation

Reply:
Yes, will do that 👍